openapi/components/schemas/history/200-get-sent-messages-request-event.yaml

```diff
@@ -0,0 +1,15 @@
+type: object
+properties:
+  messagesRequestEvents:
+    type: array
+    items:
+      type: object
+      properties:
```

Comment:
is there any of those properties required?

Reply:
Yes, all of them. Should I add this?: 
```
required:
  - friendshipId
  - actingUser
  - timestamp
  - message
```
I haven't seen that in the other endpoints though

Reply:
Yes, please. If the fields are required, then the property `required: true` needs to be set